src/content/developers/tutorials/downsizing-contracts-to-fight-the-contract-size-limit/index.md

```diff
@@ -146,3 +146,34 @@ function doSomething() { checkStuff(); }
 ```
 
 Those tips should help you to significantly reduce the contract size. Once again, I cannot stress enough, always focus on splitting contracts if possible for the biggest impact.
+
+### Use `bytes32` {#use-bytes32}
+
+- If you can fit your data in 32 bytes, then you should use `bytes32` datatype rather than bytes or strings, as it is much cheaper in Dolidity.
```

Comment:
```suggestion
- If you can fit your data in 32 bytes, then you should use `bytes32` datatype rather than bytes or strings, as it is much cheaper in Solidity.
```